includes/Modules/Analytics/Advanced_Tracking/Plugin_Detector.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Analytics\Advanced_Tracking\Plugin_Detector
+ *
+ * @package   Google\Site_Kit\Modules\Analytics
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+// phpcs:disable WordPressVIPMinimum.Constants.ConstantString.NotCheckingConstantName
+// phpcs:disable WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase
+// phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+
+namespace Google\Site_Kit\Modules\Analytics\Advanced_Tracking;
+
+/**
+ * Class for detecting the user's current active plugins that Advanced_Tracking class supports.
+ *
+ * @since n.e.x.t.
+ * @access private
+ * @ignore
+ */
+class Plugin_Detector {
+
+	/**
+	 * A list of AdvancedTracking supported plugins.
+	 *
+	 * @since n.e.x.t.
+	 * @var array
+	 */
+	private $supported_plugins = array();
+
+	/**
+	 * The constant check_type string for support_plugins array in the Advanced_Tracking class.
+	 *
+	 * @since n.e.x.t.
+	 * @var string
+	 */
+	public static $TYPE_CONSTANT = 'CONSTANT';
+
+	/**
+	 * The function check_type string for support_plugins array in the Advanced_Tracking class.
+	 *
+	 * @since n.e.x.t.
+	 * @var string
+	 */
+	public static $TYPE_FUNCTION = 'FUNCTION';
```

Comment:
Constants shouldn't start in a `$`, that can be omitted.

Reply:
Solved.